csrankings-d.csv

```diff
@@ -912,6 +913,7 @@ Devorah Kletenik,CUNY,http://www.sci.brooklyn.cuny.edu/~kletenik,cYTgpmEAAAAJ
 Dexter Campbell Kozen,Cornell University,http://www.cs.cornell.edu/~kozen,LV1qWjgAAAAJ
 Dexter Kozen,Cornell University,http://www.cs.cornell.edu/~kozen,LV1qWjgAAAAJ
 Deying Li,Renmin University of China,http://info.ruc.edu.cn/academic_professor.php?teacher_id=60,NOSCHOLARPAGE
+Deyu Meng,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/dymeng,an6w-64AAAAJ

```

Comment:
Math - not clear how this person meets the inclusion criteria. Please clarify.

Reply:
Removed.